### PR TITLE
GH-788: Access rights for announcements manager

### PR DESCRIPTION
**Pull request by tamarayustian**

### CODE REVIEW

File: ui/src/components/MainContainer.js [hunk 1]
```diff
@@ -290,6 +289,7 @@ const MainContainer = () => {
           permissions={['t3ch', 'admin', 'stewardship']}
           component={AdminTestimonyContainer}
         />
+        QWW
```
Comment (bhryck) [inline ui/src/components/MainContainer.js:292]: whats this

File: server/api/models/User.js [hunk 2]
```diff
@@ -22,7 +22,16 @@ email status until they click the link in the confirmation email.`,
     },
     accessType: {
       type: 'string',
-      isIn: ['stewardship', 'admin', 'alumni', 't3ch', 'signed', 'unsigned'],
+      isIn: [
+        'stewardship',
+        'admin',
+        'alumni',
```
[the remark this answers is not in the document]
Comment (tamarayustian) [reply server/api/models/User.js:28]: it was in our very first initial plan to update our HMCC alumnis with something. it was an ET dream request i believe

File: ui/src/components/MainContainer.js (hunk 1 above)
Comment (tamarayustian) [reply ui/src/components/MainContainer.js:292]: oops typo

File: server/api/models/User.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (bhryck) [reply server/api/models/User.js:28]: Cant really think of a scenario where alumni would have different access level than signed. Maybe a different field would be more appropriate? Regardless were not using it right now.

**Review by bhryck — APPROVED** [review]

File: server/api/models/User.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (tamarayustian) [reply server/api/models/User.js:28]: hmmm yea possibly. but still dont know yet. i don't think we will implement this anytime soon

**Review by ksuhartono97 — APPROVED** [review]

**Pull request merged**